app/views/docs/databases-relationships.phtml

```diff
@@ -0,0 +1,137 @@
+            <td>If a document has related documents, when it is deleted, the related documents are <b>also</b> deleted.</td>
+        </tr>
+        <tr>
+            <td><code>Set null</code></td>
+            <td>If a document has related documents, when it is deleted, the related documents are kept with their relationship attribute set to <b>null</b>.</td>
+        </tr>
+    </tbody>
+</table>
+
+<h2><a href="/docs/databases-relations#create-in-console" id="create-in-console">Creating Relationships in the Appwrite Console</a></h2>
+<p>You can create relationships in the Appwrite Console by adding a relationship attribute.</p>
+<ol class="margin-top margin-bottom-large text-size-normal">
+    <li>In your project, navigate to <b>Databases</b> > <b>Select your database</b> > <b>Select your collection</b> > <b>Attributes</b> > <b>Create attribute</b>.</li>
+    <li>Select <b>Relationship</b> as the attribute type.</li>
+    <li>In the <b>Relationship</b> modal, select the <a href="/docs/databases-relations#types">relationship type</a> and pick the related collection and attributes.</li>
+    <li>Pick attribute key(s) to represent the relationship. Attribute keys are used to reference a relationship in queries, so pick something that's intuitive and easy to remember.</li>
+    <li>Select desired behavior <a href="/docs/databases-relations#on-delete">on delete</a>.</li>
+    <li>Click the <b>Create</b> button to create the relationship.</li>
+</ol>
+
+<h2><a href="/docs/databases-relations#create-in-code" id="create-in-code">Creating Relationships Programmatically</a></h2>
+[TODO: Steps to create relationships in code]
+
+<h2><a href="/docs/databases-relations#create-documents" id="create-documents">Creating Documents</a></h2>
+<p>If a collection has relationship attributes, you can create documents in two ways.</p>
+
+<h3>Created Child Documents</h3>
+<p>You can create both the <b>parent</b> and <b>child</b> at once in a relationship by nesting data.</p>
+[CODE EXAMPLES]
+
+<p>If a nested document is included and <b>no conflicting document ID</b> exists, the child document will be <b>created</b>.</p>
+<p>If a nested document is included and the <b>document ID already exists</b>, the child document will be <b>updated</b>.</p>
+
+<h3>Reference Child Documents</h3>
+<p>You can create the parent and <b>reference the child documents</b> using their IDs.</p>
+[CODE EXAMPLES]
+
+<h2><a href="/docs/databases-relations#query" id="query">Queryings</a></h2>
+[TODO: how to query relationships, indexes, etc.]
+<p>You can query documents in a relation by referencing the relationship attribute key using dot-notation. For example, querying for the <code>name</code> <code>customer</code> relationship would look like <code>Query.equal('customer.name', ['John'])</code>.</p>
+
+[CODE EXAMPLES]
+<h2><a href="/docs/databases-relations#update-and-delete" id="update-and-delete">Updaing and Deleting Documents</a></h2>
+[TODO: how to query relationships, indexes, etc.]
```

Comment:
@abnegate A code example to update documents in a relationship and to delete documents in the relationship.

Code examples for Web, Flutter, Android, Apple